package.json

```diff
@@ -63,7 +63,7 @@
     }
   },
   "dependencies": {
-    "focus-lock": "^0.5.4",
-    "vue-demi": "^0.12.1"
+    "focus-lock": "0.*",
```

Comment:
This variant is highly discouraged as it might match lower version that "expected" in case you've got focus-lock already. Not a case for focus-lock but can be a problem in general, this is why discouraged.